### ISSUE TITLE
Brand tasking manger 

### ISSUE THREAD

**Issue by thomas-maschler** [crowdcover/logging-roads#7]
Under the current setup, tasking manger site should look like micro site
Change font, color and header and add logos

**Comment by jameslaneconkling** [comment]
Yeah.  Given scope and timeline, I'm going to aim to simply add project logo/masthead.

Rather than more advanced redesigning, I'd prioritize #5 , which would make the TM branding moot (users wouldn't actually visit TM page anymore)
